src/Integration/Symfony/Bundle/tests/Unit/DependencyInjection/ConfigurationTest.php

```diff
@@ -39,6 +39,7 @@ public function testDefaultValues(): void
                     'pool' => 'cache.system',
                     'ttl' => 600,
                 ],
+                'max_results' => null,
```

Comment:
Are you sure that `null`, won't be sent in the request (and AWS won't reject the payload)?

Reply:
According to the `GetParametersByPathRequest` it should not be a problem. `MaxResults` is null by default.

https://github.com/async-aws/aws/blob/125a4b95d49fff611f24ce7f6150ba4528727bd8/src/Service/Ssm/src/Input/GetParametersByPathRequest.php#L88

So I guess SSMClient already manage it correctly no ?